rust_build_utils/linux_build_utils.py

```diff
@@ -0,0 +1,40 @@
+from os import path
+import rust_build_utils.rust_utils as rutils
+from rust_build_utils.rust_utils_config import GLOBAL_CONFIG
+
+
+def strip(project: rutils.Project, config: rutils.CargoConfig, packages=None):
```

Comment:
Looks to me like it could be combined with android, no?

Reply:
Or it just over complicates things ?

Reply:
It only differs the strip tool binary and the archs. It could be combined yes, I would prefer to leave it separated as doesn't cause any overhead and it might simplify future changes. Wdyt?